docs/reference/create-client.txt

```diff
@@ -2064,6 +2064,12 @@ The following is a list of feature flags that the Mongo Ruby Driver provides:
 
    * - Flag
      - Description
+   * - ``broken_view_options``
+     - When this flag is turned off, the view options will be correctly
+       propagated to the ``aggregate``, ``count``, ``count_documents``,
+       ``distinct``, and ``estimated_document_count`` mehods. When this flag is
+       switched on, the view options will be ignored in those methods.
+       (default: false)
```

Comment:
Default should be true to have same behavior as 2.17 right?

Reply:
yes, default should say true